api/router.go

```diff
@@ -13,6 +13,7 @@ func Base(router *gin.Engine) {
 	router.GET("/boleto/confirmation", confirmation)
 	router.POST("/boleto/confirmation", confirmation)
 	router.GET("/healthcheck", healthcheck.ExecuteOnAPI)
+	router.GET("/softHealthCheck", healthcheck.SoftHealthCheck)
```

Comment:
Sugestão para tu avaliar, me parece que faz mais sentido ping, com o novo objetivo 😀 
```suggestion
	router.GET("/ping", healthcheck.ping)
```